SandBox/TwoMoment_OrderV/InitializationModule_Neutrinos.F90

```diff
@@ -77,9 +91,10 @@ SUBROUTINE InitializeFields_Relaxation
     REAL(DP), PARAMETER :: V_u_1 = 0.0_DP * SpeedOfLight
     REAL(DP), PARAMETER :: V_u_2 = 0.0_DP * SpeedOfLight
     REAL(DP), PARAMETER :: V_u_3 = 0.0_DP * SpeedOfLight
+    REAL(DP), PARAMETER :: Mu_0  = - 1.0_DP ! \in [-1,1]
```

Comment:
@eendeve What do we want these settings to be by default? Should we use your suggested `V_u_1 = 0.1` and `Mu_0 = 0`?